.github/workflows/nodejs.yml

```diff
@@ -56,10 +56,10 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
         node-version: [10.x, 12.x, 14.x, 16.x]
         webpack-version: [4, latest]
-        dev-server-version: [latest, next]
+        dev-server-version: [version-3, latest]
```

Comment:
let's just remove v3 snapshots? They are already tested

Reply:
We will drop `webpack-dev-server@3` support for `webpack-cli@next`

Reply:
let's drop them in next then